hgtv/views/video.py

```diff
@@ -39,21 +39,25 @@ def process_video(video, new=False):
         if parsed.netloc in ['youtube.com', 'www.youtube.com']:
             try:
                 video_id = parse_qs(parsed.query)['v'][0]
-                r = requests.get('https://gdata.youtube.com/feeds/api/videos/%s?v=2&alt=json' % video_id)
-                jsondata = r.json() if callable(r.json) else r.json
-                if jsondata is None:
+                r = requests.get('https://www.googleapis.com/youtube/v3/videos?part=snippet&id={video_id}&key={api_key}'.format(
+                    video_id=video_id, api_key=app.config.get('YOUTUBE_API_KEY', '')
```

Comment:
Use `api_key=app.config['YOUTUBE_API_KEY']`. The call will fail if there's no key anyway. Let this failure propagate as a server error rather than an ambiguous error message to the user.